README.md

```diff
@@ -467,6 +467,26 @@ Trade when % change from last two 1m periods is higher than average.
 - In a market with predictable price surges and corrections, `--profit_stop_enable_pct=10` will try to sell when the last buy hits 10% profit and then drops to 9% (the drop % is set with `--profit_stop_pct`). However in strong, long uptrends this option may end up causing a sell too early.
 - For Kraken and GDAX you may wish to use `--order_type="taker"`, this uses market orders instead of limit orders. You usually pay a higher fee, but you can be sure that your order is filled instantly. This means that the sim will more closely match your live trading. Please note that GDAX does not charge maker fees (limit orders), so you will need to choose between not paying fees and running the risk orders do not get filled on time, or paying somewhat high % of fees and making sure your orders are always filled on time.
 
+## Notifiers
+
+Zenbot employs various notifiers to keep you up to date on the bot's actions. We currently send a notification on a buy and on a sell signal.
+
+### pushbullet
+
+Supply zenbot with your api key and device ID and we will send your notifications to your device.
+
+### Slack
+
+Supply zenbot with a webhook URI and zenbot will push notifications to your webhook.
+
+### XMPP
+
+Supply zenbot with your XMPP credentials and zenbot will send notifications by connecting to your XMPP, sending the notification, and disconnecting.
+
+### IFTTT
+
+Supply zenbot with your IFTTT maker key and zenbot will push notifications to your IFTTT.
```

Comment:
here the service has been integrated using webhooks of IFTTT: https://ifttt.com/maker_webhooks